package.json

```diff
@@ -40,6 +40,7 @@
     "@types/react-modal": "^3.10.6",
     "@types/util-deprecate": "^1.0.0",
     "classnames": "^2.2.6",
+    "copy-to-clipboard": "^3.3.1",
```

Comment:
Out of interest - any reason why not to use a hook that does this already? e.g. https://github.com/danoc/react-use-clipboard - saves a bit of code - haven't checked the size difference but I suspect they will be pretty similar in size

Reply:
No specific reason. Happy to move to this. Looking at the code its exactly same as I had